### PR TITLE
Upgrade to Node.js 20.11.0 LTS

### PR DESCRIPTION
**Pull request by Juneezee**
# Description and Motivation

Closes https://github.com/SatcherInstitute/health-equity-tracker/issues/2654.

## Has this been tested? How?

- `npm run build`
- `npm run test`

## Types of changes

- Refactor / chore

### CODE REVIEW

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -24,9 +24,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: 19
+          node-version-file: frontend/package.json
```
Comment (Juneezee) [inline .github/workflows/frontend.yml:29]: Automatically detect the `node` version that we set in our `frontend/package.json`, see https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

In the future, we only need to update the `node` version in `package.json`.

File: frontend/package.json [hunk 2]
```diff
@@ -127,8 +127,8 @@
     "url": "npx playwright install chromium && npx playwright test --project=URL"
   },
   "engines": {
-    "node": ">=19",
-    "npm": ">=6.14.8"
+    "node": ">=20.11.0",
+    "npm": ">=10.2.4"
```
Comment (Juneezee) [inline frontend/package.json:131]: image

https://nodejs.org/en/about/previous-releases#looking-for-latest-release-of-a-version-branch

File: .github/workflows/frontend.yml (hunk 1 above)
Comment (benhammondmusic) [reply .github/workflows/frontend.yml:29]: oh nice!

**Review by benhammondmusic — APPROVED** [review]
fantastic

**Pull request merged**